interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -2224,18 +2234,23 @@ public function getCarecoordinationModuleSettingValue($field_name)
     public function getDetails($field_name)
     {
         if ($field_name == 'hie_custodian_id') {
-            $query = "SELECT f.name AS organization, f.street, f.city, f.state, f.postal_code AS zip, f.phone AS phonew1
-            FROM facility AS f
-            JOIN modules AS mo ON mo.mod_directory='Carecoordination'
-            JOIN module_configuration AS conf ON conf.field_value=f.id AND mo.mod_id=conf.module_id
-            WHERE conf.field_name=?";
-        } else {
-            $query = "SELECT u.title, u.fname, u.mname, u.lname, u.npi, u.street, u.city, u.state, u.zip, CONCAT_WS(' ','',u.phonew1) AS phonew1, u.organization, u.specialty, conf.field_name, mo.mod_name, lo.title as  physician_type, SUBSTRING(lo.codes, LENGTH('SNOMED-CT:')+1, LENGTH(lo.codes)) as  physician_type_code
-            FROM users AS u
+            $query = "SELECT f.name AS organization, f.street, f.city, f.state, f.postal_code AS zip, f.phone, f.uuid AS phonew1
+        FROM facility AS f
+        JOIN modules AS mo ON mo.mod_directory='Carecoordination'
+        JOIN module_configuration AS conf ON conf.field_value=f.id AND mo.mod_id=conf.module_id
+        WHERE conf.field_name=?";
+        } elseif (is_string($field_name)) {
+            $query = "SELECT u.title, u.fname, u.mname, u.lname, u.npi, u.street, u.city, u.state, u.zip, CONCAT_WS(' ','',u.phonew1) AS phonew1, u.organization, u.specialty, conf.field_name, mo.mod_name, lo.title as  physician_type, SUBSTRING(lo.codes, LENGTH('SNOMED-CT:')+1, LENGTH(lo.codes)) as  physician_type_code, uuid
+        FROM users AS u
+        LEFT JOIN list_options AS lo ON lo.list_id = 'physician_type' AND lo.option_id = u.physician_type
+        JOIN modules AS mo ON mo.mod_directory='Carecoordination'
+        JOIN module_configuration AS conf ON conf.field_value=u.id AND mo.mod_id=conf.module_id
+        WHERE conf.field_name=?";
+        } elseif (is_int($field_name)) {
```

Comment:
If I'm following this correctly, you treat the field_name as the user id if its an integer?

Reply:
yep